components/accelerator/nvidia/query/infiniband/infiniband.go

```diff
@@ -12,15 +12,6 @@ import (
 	"github.com/leptonai/gpud/pkg/process"
 )
 
-// Returns true if the product supports infiniband.
-// e.g.,
-// "NVIDIA A100"
-// "NVIDIA H100"
-func SupportsInfinibandProduct(gpuProductName string) bool {
-	p := strings.ToLower(gpuProductName)
-	return strings.Contains(p, "a100") || strings.Contains(p, "h100")
-}
-
 // Returns the default non-zero per-port rate in GB/sec if the product supports infiniband.
 func SupportsInfinibandPortRate(gpuProductName string) int {
```

Comment:
can we remove this func as well? which part still consumes it?

Reply:
Let me remove, was going to do in a separate PR

Reply:
Keeping it for "gpud scan" in https://github.com/leptonai/gpud/pull/380.